_posts/2017-04-27-Rust-1.17.md

```diff
@@ -0,0 +1,352 @@
+---
+layout: post
+title: "Announcing Rust 1.17"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.17.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed, getting Rust 1.17 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.17.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1170-2017-04-27
```

Comment:
Note that for at least the posts announcing Rust 1.0 to 1.3, the "release notes" links are to /master/ and the linked anchor no longer exists on the page so the user has to scroll to find the right spot.  The Rust 1.4 post links to release notes with a /[commit-id]/ link instead, so the outdated anchor tag format still works.

Consider either updating these links after publication to each point at a specific version of the release notes file, or updating the now broken links in those posts.